src/driver/main.cpp

```diff
@@ -578,6 +578,12 @@ struct verify : command<verify>
     }
 };
 
+struct ort_sha : command<ort_sha>
+{
+    void parse(const argument_parser&) {}
+    void run() const { std::cout << MIGRAPHX_ORT_SHA1 << std::endl; }
```

Comment:
This should be moved to the main function. As ort_sha shouldn't show up in the help.

Reply:
Done, removed